AjuntamentdeBarcelona/decidim.barcelona
# Add cookies warning

# TODOs
- [x] Cookies warning component
- [ ] @josepjaume css magic tricks
# What and why

In order to have the EU ePrivacy Directive compliance I added a cookies warning the first time you visit the website.
# QA

Load the page and see the warning message at the top. Refresh the page and the message should disappear.
# GIF tax

![](https://media.giphy.com/media/w2YGvWXRzFM52/giphy.gif)
